icons/captions-off.json

```
@@ -0,0 +1,18 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "defaultlp"
```

Your name should be exactly the same as in you Github URL like. So it should be:
```suggestion
    "DefaultLP"
```